src/int_utils.h

```diff
@@ -130,7 +132,11 @@ constexpr inline I Mask() { return ((I((I(-1)) << (std::numeric_limits<I>::digit
 /** Compute the smallest power of two that is larger than val. */
 template<typename I>
 static inline int CountBits(I val, int max) {
-#ifdef _MSC_VER
+#if defined(__cpp_lib_bitops)
```

Comment:
Shouldn't the [`__cpp_lib_int_pow2`](https://en.cppreference.com/w/cpp/feature_test#cpp_lib_int_pow2) macro be checked instead?

Reply:
Nice catch, thanks! This was originally implemented in terms of `countl_zero` and I forgot to update it after the switch.

Reply:
Updated to use the correct macro.

Also, it turns out that some stdlib versions defined the macro before providing `std::bit_width`, so I've added the proper check for version as well.